p2p/rating/peersRatingHandler.go

```diff
@@ -186,6 +192,29 @@ func (prh *peersRatingHandler) GetTopRatedPeersFromList(peers []core.PeerID, min
 	return peersTopRated
 }
 
+func (prh *peersRatingHandler) displayPeersRating(peers *[]core.PeerID, minNumOfPeersExpected int) {
+	if log.GetLevel() != logger.LogTrace {
+		return
+	}
+
+	strPeersRatings := ""
+	for _, peer := range *peers {
+		rating, ok := prh.topRatedCache.Get(peer.Bytes())
+		if !ok {
+			rating, _ = prh.badRatedCache.Get(peer.Bytes())
+		}
+
+		ratingInt, ok := rating.(int32)
+		if ok {
+			strPeersRatings += fmt.Sprintf("\n peerID: %s, rating: %d", peer.Pretty(), ratingInt)
+		} else {
+			strPeersRatings += fmt.Sprintf("\n peerID: %s, rating: invalid", peer.Pretty())
+		}
+	}
+
+	log.Info("Top rated peers", "min requested", minNumOfPeersExpected, "peers ratings", strPeersRatings)
```

Comment:
log.Trace

Can we also print the bad peers?

Reply:
fixed log.Trace.. the list includes also bad peers, I updated the log message which was confusing